app/controllers/EmailController.scala

```diff
@@ -39,11 +39,13 @@ class EmailController @Inject()(authenticate: IdentifierAction,
   val log: LoggerLike = Logger(this.getClass)
 
   def showUnverified():Action[AnyContent] = authenticate async { implicit request =>
```

Comment:
Please format this method as the indentation at line no 41 and 44 is not correct